client/src/components/newColumnArea/ColumnArea.js

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback, useContext } from "react";
+import Column from "../newColumn/Column";
+import { makeStyles } from "@material-ui/core/styles";
+import { UserContext } from "../../userContext/userContext";
+import { DragDropContext } from "react-beautiful-dnd";
+
+const ColumnArea = props => {
+  const classes = useStyles(props);
+  const { value1 } = useContext(UserContext);
+  const [taskState, setTaskState] = value1;
+
+  const onDragEnd = () => {
+    //ToDo reordering logic
+  };
+
+  return (
+    <div className={classes.root}>
+      <DragDropContext>
+        {taskState.map((column, index) => (
+          <Column
+            key={column.columnId}
```

Comment:
could send this in as a string so you dont have to call toString() in the Column component. new way is just String(column.columnID)

Reply:
Thanks. I fixed!